compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3178,6 +3178,7 @@ public void visit(BLangServiceConstructorExpr serviceConstructorExpr, AnalyzerDa
     @Override
     public void visit(BLangQueryExpr queryExpr, AnalyzerData data) {
         data.queryToTableWithKey = queryExpr.isTable() && !queryExpr.fieldNameIdentifierList.isEmpty();
+        data.queryToMap = queryExpr.isMap;
```

Comment:
What is this change?

Reply:
This is required to skip the check on L3264 in CodeAnalyzer.java.